sysdig/resource_sysdig_secure_drift_policy_test.go

```diff
@@ -138,16 +139,17 @@ resource "sysdig_secure_drift_policy" "sample" {
     description = "Test Drift Rule Description"
 
     enabled = true
+    use_regex = true
```

Comment:
Same here, do not add it to a previous existing resource tested. Add a new test case for the `use_regex = true`.